### DISCUSSION TITLE
birdnetpi.com might have a certificate or ownership issue as of today (30-Apr-2023)

### DISCUSSION THREAD

**Discussion by Astrobirder** [mcguirepr89/BirdNET-Pi#882] [General]
All, 

I just noticed that I can't visit https://birdnetpi.com any longer.  I'm pretty sure I've been there over the last few days. (History search says I was successful on 27-Apr-2023), but today I get:

ERR_SSL_VERSION_OR_CIPHER_MISMATCH

The http:// non-ssl version shows Domain Reserved via Strato.  Not sure who exactly owns this, but [WHOIS reports](https://www.whois.com/whois/birdnetpi.com) a change was made yesterday 29-APR-2023.

As I didn't know who needed to know this, if anyone, I thought I'd post here.

**Comment by Svardsten53** [comment]
If I remember correctly, Patrick bought this domain for general information about BirdNET-PI, but also so that we could have our own installations as a subdomain of birdnetpi.com. If the shutdown is permanent, we who have subdomains there should remove this from our caddyfile.

**Comment by ehpersonal38** [comment]
The website is currently being transfered to a new owner, it will hopefully be up and running soon. I'm not sure if the subdomain DNS records will be preserved.

**Reply by jds3d** [reply]
Is there any update on this?  I just recorded a bay breasted warbler and want to share.

**Reply by oliverk71** [reply]
The domain points to the github page, now. So, ownership does not seem to be an issue. 
However, since the subdomains do not work, I think it would be best to use a dyndns domain, at least as a fallback solution. Maybe in combination with a top level domain.
